### PR TITLE
Ddf 2481 2.9.x update Windows installation documentation

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Removes unneeded step to add user on Windows. (2.9.x version)

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?

@AzGoalie @Lambeaux @spearskw 

#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).

[Docs](https://github.com/orgs/codice/teams/docs)

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@figliold
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

verify command removed.

#### What are the relevant tickets?
[DDF-2481](https://codice.atlassian.net/browse/DDF-2481)

#### Checklist:
- [x] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/installing-contents.adoc [hunk 1]
```diff
@@ -128,10 +128,9 @@ chown ${branding-lowercase}-user:${branding-lowercase}-group new_installation
 su - ${branding-lowercase}-user
 ----
 +
-.Example: Create a Directory on Windows and Switch User on Windows
+.Example: Create a Directory on Windows(Switching user not required for Windows)
```
Comment (jrnorth) [inline distribution/docs/src/main/resources/_contents/installing-contents.adoc]: Space before `(`?

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/installing-contents.adoc]: fixed.

**Comment by taz77** [comment]
Looks good. These were confusing steps in the FedFest.

**Comment by cruzj6** [comment]
Looks good to me, glad the FedFest caught this stuff.

**Pull request merged**